gen/definitions/classes.yaml

```diff
@@ -154,16 +155,23 @@ fvFBRGroup:
 
 fvFBRMember:
   resource_name: "vrf_fallback_route_group_member"
-  contained_by:
-    - "fvFBRGroup"
   ui_locations:
     - "Tenants -> Networking -> VRFs -> Policy -> Fallback Route Group -> Fallback Members"
   sub_category: "Networking"
   resource_notes:
     - "This resource should not be used in combination with the `vrf_fallback_route_group` nested attributes of other resources for the same object. Doing so will result in unexpected behaviour."
 
+fvFBRoute:
+  resource_name: "vrf_fallback_route"
+  max_one_class_allowed: true
```

Comment:
Why was max_one_class_allowed introduced?
I'm asking because going forward if we use this in other resources, it will be applied to all the children and not jut the one we intend to target?

Reply:
After having a discussion- this setting is consumed at the class level.